crates/guest/util/identity/Cargo.toml

```diff
@@ -7,3 +7,4 @@ edition = "2021"
 
 [dependencies]
 risc0-zkvm = { version = "1.2", default-features = false, features = ["std"] }
+risc0-zkvm-platform = { version = "1.2", features = ["sys-getenv"] }
```

Comment:
This makes the `identity` guest insecure and vulnerable to dev-mode attacks. I don't think it's a problem, as this is just a test image, but it's definitely not great.